src/components/forms/fragments/ExchangeApplication/index.tsx

```diff
@@ -0,0 +1,3 @@
+import ExchangeApplication from './ExchangeApplication';
```

Comment:
Didn't we agree not to use index files? ;)

Reply:
@rBoost: well at least its better than `./ExchangeApplication/ExchangeApplication` import to me

Reply:
well, the folder name should be `exchangeApplication` so import would be `./exchangeApplication/ExchangeApplication`.

Reply:
only component files should have ucfirst